docs/how-to-guides/add-a-custom-ros-message.md

```diff
@@ -0,0 +1,100 @@
+# Add a custom ROS message
+
+## Description
+
+During the Autoware development, you will probably need to define your own messages, This page lists a simple example of adding a custom message to Autoware, For the general ROS2 tutorial see [Create custom msg and srv files](http://docs.ros.org/en/galactic/Tutorials/Beginner-Client-Libraries/Custom-ROS2-Interfaces.html).
+
+## How to create custom message
+
+Make sure you are in the Autoware workspace, and then run the following command to create a new package.
+
+For example we create a package to define sensor messages.
+
+1. Create a package
+
+   ```console
+   ros2 pkg create --build-type ament_cmake autoware_sensing_msgs
+   ```
+
+1. Create custom messages
```

Comment:
Could you use the ordered style?
```suggestion
2. Create custom messages
```
Related: https://github.com/autowarefoundation/autoware/pull/3006